tutorials/gSlack/index.md

```diff
@@ -0,0 +1,172 @@
+
+Make sure your `gcloud` SDK and CLI environment is authenticated against the requested GCP project with a user that has Owner permissions.
+
+### 1) Create a topic 
+Run the following command to create a `Google Cloud PubSub` topic:
+```
+$ gcloud --project=<YOUR_PROJECT_ID> beta pubsub topics create gcp-alert-service
+```
+The command will take a few seconds to run. 
+Once complete your project will contain a new topic that you can see at the [Google Cloud PubSub Console](https://console.cloud.google.com/cloudpubsub/topicList).
+We will use this topic as a sink to which log exports would write our events.
+
+### 2) Create an export sink
+Run the following command to create the export sink:
+```
+$ gcloud --project=<YOUR_PROJECT_ID> beta logging sinks create gcp_alert_service pubsub.googleapis.com/projects/<YOUR_PROJECT_ID>/topics/gcp-alert-service --log-filter "logName=projects/<YOUR_PROJECT_ID>/logs/cloudaudit.googleapis.com%2Factivity"
+```
+
+This command creates a log export that uses the specified filter to send all `Cloud Audit Logging` activity events into the topic we created before.
+
+You can see the new export sink at the [Log Exports Console](https://console.cloud.google.com/logs/exports).
+Notice the warning icon attached to it.
+
+As part of creating the export sink - a new service account was created. This service account is used to publish event into our topic.
+However this service account still lacks the permissions required to publish into `Google PubSub`.
+
+### 3) Granting service account permissions
+Run the following command:
+```
+$ gcloud --project=<YOUR_PROJECT_ID> projects add-iam-policy-binding <YOUR_PROJECT_ID> --member=$(gcloud --project <YOUR_PROJECT_ID> --format="value(writer_identity)" beta logging sinks describe gcp_alert_service) --role='roles/pubsub.publisher'
+```
+This command grants a `pubsub.publisher` role to the service account that was created previously.
+A few minutes after this command finishes you should see the warning icon removed from the export sink in the [Log Exports Console](https://console.cloud.google.com/logs/exports).
+
+## Deploying the code
+At this stage we have a export log that pushes all audit events into the topic we created.
+To complete the application we need some code to recieve these events, filter them and use the `Slack` API to post messages on the desired `Slack` channel.
+One way to do so that uses a fully manaaged, auoscalable, serveless comupting service is by using [Google CloudFunctions](https://cloud.google.com/functions/).
+This service will deploy and scale our code as much as required for optmial performance.
+
+`Google CloudFunctions` use [Node.js](https://nodejs.org/en/) as a runtime platform which means our code will be written using Javascript.
+
+Before we can make a `Google CloudFunctions` deployment we must create a `Google Cloud Storage` bucket for staging and enable the `Google CloudFunctions API` for our project.
```

Comment:
Add a comma:
Before we can make a \`Google CloudFunctions\` deployment`,` we must...